src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -259,7 +236,7 @@ function DatasetOverviewPanel(props) {
                         </dt>
                       </dl>
                       {i < dataset.schema.relationships.length - 1 && (
-                        <Divider className={classes.divider} />
+                        <Divider sx={{ mt: '14px', mb: '14px' }} />
```

Comment:
Same with `mt` and `mb` - I'd prefer the longer versions even if that is functional shorthand.